gui/functionality/idact_app.py

```diff
@@ -24,7 +23,7 @@ def __init__(self):
         self.remove_argument_window = RemoveArgumentApp()
         self.show_native_arguments_window = ShowNativeArgumentsApp()
         self.parameters = self.saver.get_map()
-        self.successWindow = SuccessWindow()
+        self.popUpWindow = PopUpWindow()
 
 
 class AddArgumentApp(QMainWindow):
```

Comment:
This is not an `App` rename to `Window`. The same for `RemoveArgumentApp` and `ShowNativeArgumentsApp`